packages/x-tailwindcss/src/x-tailwind-plugin/components/sliding-panel/buttons/hover.ts

```diff
@@ -21,18 +34,6 @@ export function slidingPanelButtonsHover() {
           pointerEvents: 'all'
         }
       }
-    },
-    '&:not(.sliding-panel-show-buttons-on-hover):not(.sliding-panel-at-start)': {
-      '.sliding-panel-button-left': {
-        opacity: 1,
-        pointerEvents: 'all'
-      }
-    },
-    '&:not(.sliding-panel-show-buttons-on-hover):not(.sliding-panel-at-end)': {
-      '.sliding-panel-button-right': {
-        opacity: 1,
-        pointerEvents: 'all'
-      }
```

Comment:
After removing these rules,`pointerEvents` value is always `none` for Sliding Panels that have the "show buttons on hover" feature disabled, so the buttons are not clickable.